shared-libs/cht-datasource/src/person.ts

```diff
@@ -32,14 +32,23 @@ export namespace v1 {
     }
   };
 
-  const assertTypeQualifier: (
+  const assertTypeQualifier: (qualifier: unknown) => asserts qualifier is ContactTypeQualifier = (
     qualifier: unknown
-  ) => asserts qualifier is ContactTypeQualifier = (qualifier: unknown) => {
+  ) => {
     if (!isContactTypeQualifier(qualifier)) {
       throw new Error(`Invalid type [${JSON.stringify(qualifier)}].`);
     }
   };
 
+  const assertLimitAndSkip = (limit: unknown, skip: unknown) => {
```

Comment:
Minor, but for the sake of consistency, I think we should just break this into two assert functions.